src/internal/OpenApiTools/components/Reference.ts

```diff
@@ -136,22 +136,29 @@ export const generate = <T>(entryPoint: string, currentPoint: string, reference:
     throw new NotFoundFileError(`Not found reference point from current point. \n Path: ${referencePoint}`);
   }
 
-  const relativePathFromEntryPoint = path.relative(path.dirname(entryPoint), referencePoint); // components/hoge/fuga.yml
-  const ext = path.extname(relativePathFromEntryPoint); // .yml
-  const pathArray: string[] = relativePathFromEntryPoint.replace(ext, "").split(path.sep); // ["components", "hoge", "fuga"]
-  const targetPath: string = pathArray.join("/"); // components/hoge/fuga
+  const fragmentIndex = referencePoint.indexOf("#/");
+  let targetPath: string;
+  if (fragmentIndex !== -1) {
+    targetPath = referencePoint.substring(fragmentIndex + 2);
+  } else {
+    const relativePathFromEntryPoint = path.relative(path.dirname(entryPoint), referencePoint); // components/hoge/fuga.yml
+    const pathArray: string[] = relativePathFromEntryPoint.split(path.sep); // ["components", "hoge", "fuga"]
```

Comment:
Since this file also does not use the posix path, CI in the Windows environment is considered to have failed.

```diff
- import * as path from "path";
+ import { posix as path } from "path";
```

Reply:
This is another problem ...
https://github.com/Himenon/openapi-typescript-code-generator/pull/87